unit_test/test_spdm_callback/README.md

```diff
@@ -0,0 +1,37 @@
+## This is reference implementation to verify CertChain DiceTcbInfo extension.

```

Comment:
This file should be in `os_stub/spdm_cert_verify_callback_sample`.

Reply:
Thanks. I have moved the README to os_stub.